src/realmGeol.ttl

```diff
@@ -57,307 +288,1624 @@ sorelsp:hasUpperBoundary rdf:type owl:ObjectProperty .
+                                                            <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                                            <https://doi.org/10.1130/9780813774022> ;
+                                             skos:definition "Polarity Zones may be grouped into Polarity Superzones (Hyperzones in Russian literature). The corresponding interval of geological time during which the rocks in the Polarity Superzone were deposited is referred to as its associated Polarity Superchron. As an example of the latter, the mid-Carboniferous – Middle Permian Kiaman Superchron is the longest known period of predominantly reversed polarity, which was terminated by the Illawarra Geomagnetic Polarity Reversal near the base of the Capitanian. (Salvador, 2013; ICS, Online, December, 2019; Gradstein et al., 2012)"@en ;
+                                             skos:prefLabel "magnetostratigraphic polarity superzone"@en .
+
+
+###  http://sweetontology.net/realmGeol/MagnetostratigraphicPolarityZone
+soreag:MagnetostratigraphicPolarityZone rdf:type owl:Class ;
+                                        rdfs:subClassOf soreag:MagnetostratigraphicUnit ;
+                                        dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                        dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                                       <https://doi.org/10.1130/9780813774022> ;
+                                        skos:definition "The basic formal unit in magnetostratigraphic polarity classification is the Magnetostratigraphic Polarity Zone (or Polarity Zone). The corresponding interval of geological time during which the rocks in the Polarity Zone were deposited is referred to as its associated Polarity Chron. (Salvador, 2013; ICS, Online, December, 2019)"@en ;
+                                        skos:prefLabel "magnetostratigraphic polarity zone"@en .
+
+
+###  http://sweetontology.net/realmGeol/MagnetostratigraphicUnit
+soreag:MagnetostratigraphicUnit rdf:type owl:Class ;
+                                rdfs:subClassOf soreag:StratigraphicUnit ;
+                                dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                                dcterms:source <http://www.stratigraphy.org/index.php/ics-stratigraphicguide> ,
+                                               <https://doi.org/10.1130/9780813774022> ;
+                                skos:definition "A Magnetostratigraphic Unit (Magnetozone) is a body of rocks unified by similar magnetic characteristics which allow it to be differentiated from adjacent rock bodies.  The term is collectively applied to the different kinds of stratigraphic units that are identified by their measurable magnetic properties. (Salvador, 2013; ICS, Online, December, 2019)"@en ;
```

Comment:
'body of rocks' or Geologic Unit?

Reply:
The International Code, both in online and hard-copy/printed versions (e.g., Murphy & Salvador), refer to 'body of rocks' throughout.